self-core-impl/src/main/java/com/selfxdsd/core/projects/GithubWebhookEvent.java

```diff
@@ -90,6 +90,14 @@ public String type() {
             } else {
                 resolved = type;
             }
+        } else if("label".equalsIgnoreCase(type)) {
+            final String label = event.getJsonObject("label")
+                .getString("name");
+            if(label.matches("\\d+(mins|min|m)")){
```

Comment:
@amihaiemil can't remember if "60 min" is valid too besides "60min";  in that case I'll need to modify the regex to accept spaces.

Reply:
@criske Nu cred ca e nevoie de logica asta aici.

Mai bine facem logica aceasta mai generica: luam ca event orice schimbare de label si dam mai departe la PM. Vezi commentul de mai sus. 

Dar lasa un puzzle pt asta, baby steps : D